src/std/os/fdio.ss

```diff
@@ -43,6 +43,23 @@
     (do-retry-nonblock (_close raw)
       (close raw))))
 
+(def (fdflush raw)
```

Comment:
lets call this fdsync, and the underlying ffi calls accorsingly.

fsync does more tan flushing.